src/components/Layout/LayoutBottomBanner.tsx

```diff
@@ -0,0 +1,16 @@
+import { forwardRef } from 'react'
+import { cx } from 'class-variance-authority'
+
+import { LayoutBanner, type LayoutBannerProps } from './LayoutBanner'
+
+export type LayoutBottomBannerProps = LayoutBannerProps
+// @ts-ignore
+export const LayoutBottomBanner = forwardRef<HTMLDivElement, LayoutBottomBannerProps>(
```

Comment:
Do we need a `BottomBanner` ?

Reply:
It can be commented until used

Reply:
IMO I prefer to keep unused code outside of the repository,